pkgs/development/libraries/gstreamer/bad/default.nix

```
@@ -297,10 +299,11 @@ stdenv.mkDerivation rec {
     "-Ddoc=disabled" # needs gstcuda to be enabled which is Linux-only
     "-Dnvcodec=disabled" # Linux-only
     "-Dva=disabled" # see comment on `libva` in `buildInputs`
+  ] ++ lib.optionals (stdenv.isDarwin || !guiSupport) [
+    "-Ddirectfb=disabled"
```

```suggestion
  ] ++ lib.optionals (!stdenv.isLinux || !guiSupport) [
    "-Ddirectfb=disabled"
```

Should be conditional on `stdenv.isLinux` to match the earlier `buildInput` conditional